helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -480,6 +498,7 @@ public Builder(final String hash, final Parameters parameters) {
       public String version;
       public String image;
       public String hostname;
+      public Long created;
```

Comment:
Also add something to clear this out in `withoutMetaParameters` below. That keeps it from affecting the hash.